packages/react-instantsearch-core/src/connectors/connectGeoSearch.js

```diff
@@ -128,21 +128,24 @@ export default createConnector({
   displayName: 'AlgoliaGeoSearch',
 
   getProvidedProps(props, searchState, searchResults) {
-    const results = getResults(searchResults, { ais: props.contextValue });
+    const results = getResults(searchResults, {
+      ais: props.contextValue,
```

Comment:
We can store the context inside a value rather the create the same object three times.